posthog/cdp/templates/_internal/template_error_tracking.py

```diff
@@ -0,0 +1,69 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+template_error_tracking_alert: HogFunctionTemplate = HogFunctionTemplate(
+    status="free",
+    type="error_tracking_alert",
+    id="template-error-tracking-alert",
+    name="Error Tracking Issue Alert",
+    description="",
+    icon_url="/static/services/slack.png",
+    category=["Error Tracking"],
+    filters={"trigger": "created"},
```

Comment:
I think this will need to change. We'll probably want `$$error_tracking_issue_created`